data/monster/monsters/ghulosh_deathgaze.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Ghulosh' Deathgaze" nameDescription="Ghulosh' Deathgaze" race="undead" experience="0" speed="30">
+	<health now="300000" max="300000" />
+	<look type="1063" body="57" feet="57" head="57" legs="57" addons="0" />
+	<targetchange interval="2000" chance="5" />
+	<flags>
+		<flag isboss="1" />
+		<flag pushable="0" />
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="1" />
+		<flag canwalkonfire="1" />
+		<flag canwalkonpoison="1" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="0" max="-200" />
```

Comment:
change this to:

		<attack name="death" interval="2000" chance="12" target="0" length="8" spread="0" min="-6000" max="-6000">
			<attribute key="areaEffect" value="mortarea" />
		</attack>